### ISSUE TITLE
When eating the hot dog, the motor hangs

### ISSUE THREAD

**Issue by Mac1512** [scemino/engge#211]
**Describe the bug**
When eating the hot dog, the motor hangs

**To Reproduce**

I was driving Reyes, when I finished the part of the ransome clown and reappeared in the cafeteria, the dialogue that has appeared is that of the ransome clown. When I chose the option to eat the hot dog, after Reyes vomited it, the engge motor has gone crazy, Ray appeared for a moment in the sewers, then Reyes another moment in the sewers, and finally Ray appeared in the outside the cafeteria, at which point engge has been hung.

**Expected behavior**

**Screenshots**
![fallo cafeteria](https://user-images.githubusercontent.com/2179224/85207128-04921380-b327-11ea-8e5c-13a3a5c2f933.PNG)

**Desktop (please complete the following information):**
 - OS: Windows x64
 - Version 10

**Additional context**

Tested on commit: (https://github.com/scemino/engge/commit/93ecfc3e81e05f5fadd7a8eb8da6376b247d96c2)

**Comment by scemino** [comment]
Is it possible to add a savegame in attachment to reproduce it ? Thank you.

**Comment by Mac1512** [comment]
I have not been able to reproduce this bug again, excuse me for rushing to put it on. Maybe I did some different behavior when it happened to me.

However, I pass you a saved game, since I have appreciated a possible bug, and that is that when you are in the dressing room of Ransome, if you try to go to the stage it says:

**I can't go onstage without my joke book.**

The joke book is in inventory. What I do is take the list from the tip jar (Just look at it) and he already lets me go on stage.

This error only happens to me if I load the saved game. So I do not know if it is a specific bug or if, on the contrary, it always happens when saving and loading the game at this point.

I have also observed (This always happens) that the list of swear words in the tip jar in the dressing room, if the language is Spanish, the text does not appear, but the text appears if the language is English.

When loading this save game, on at least two occasions this has appeared:

![curious_saves](https://user-images.githubusercontent.com/2179224/85235142-f1f50880-b412-11ea-8b0a-7c42639ac74b.PNG)

It's curious

Link:

https://www.mediafire.com/file/ev2zyhnvdqvsk7m/Save5.7z/file
